docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,392 @@
+
+```json
+{
+  "kubernetes":
+  {
+      "pod_name": "myPod-74db47d99-ppnsw",
+      "namespace_name": "myNamespace",
+      "pod_id": "88dbd1ef-d977-4636-804d-ef220454be1c",
+      "host": "myHost1",
+      "container_name": "myContainer",
+      "docker_id": "5649c36fcc1e956fc95e3145441f427d05d6e514fa439f4e4f1ccee80fb2c037",
+      "container_hash": "myImage@sha256:1f8d852989c16345d0e81a7bb49da231ade6b99d51b95c56702d04c417549b26",
+      "container_image": "myImage:myImageTag",
+      "labels":
+      {
+          "app": "myApp",
+          ...
+      },
+      "annotations":
+      {
+          "sidecar.istio.io/inject"=>"true",
+          ...
+      }
+  }
+}
+```
+
+### Kubernetes filter (JSON parser)
+
+After the enrichment of the log record with the Kubernetes-relevant metadata, the [Kubernetes filter](https://docs.fluentbit.io/manual/pipeline/filters/kubernetes) also tries to parse the record as a JSON document. If that was successful, all the parsed root attributes of the parsed document are added as new individual root attributes of the log.
+
+Record **before** applying the JSON parser:
+
+```json
+{
+  "time": "2022-05-23T15:04:52.193317532Z",
+  "stream": "stdout",
+  "_p": "F",
+  "log": "{\"level\": \"warn\",\"message\": \"This is the actual message\",\"tenant\": \"myTenant\",\"traceID\": \"123\"}",
+  "kubernetes": {...}
+}
+```
+
+Record **after** applying the JSON parser:
+
+```json
+{
+  "time": "2022-05-23T15:04:52.193317532Z",
+  "stream": "stdout",
+  "_p": "F",
+  "log": "{\"level\": \"warn\",\"message\": \"This is the actual message\",\"tenant\": \"myTenant\",\"traceID\": \"123\"}",
+  "kubernetes": {...},
+  "level": "warn",
+  "message": "This is the actual message",
+  "tenant": "myTenant",
+  "traceID": "123"
+}
+```
+
+### Rewrite tag
+
+As per `LogPipeline` definition, a dedicated [rewrite_tag](https://docs.fluentbit.io/manual/pipeline/filters/rewrite-tag) filter is introduced. The filter brings a dedicated filesystem buffer for the outputs defined in the related pipeline, and with that, ensures a shipment of the logs isolated from outputs of other pipelines. As a consequence, each pipeline runs on its own [tag](https://docs.fluentbit.io/manual/concepts/key-concepts#tag).
+
+## Limitations
+
+Currently there are the following limitations for LogPipelines that are served by Fluent Bit:
+
+### Fluent Bit plugins
+
+You cannot enable the following plugins, because they potentially harm the stability:
+
+- Multiline Filter
+- Kubernetes Filter
+- Rewrite_Tag Filter
+
+### Buffer limits
+
+Fluent Bit buffers up to 1 GB of logs if a configured output cannot receive logs. The oldest logs are dropped when the limit is reached.
+
+### Throughput
+
+Each Fluent Bit Pod can process up to 10 MB/s of logs for a single log pipeline. With multiple pipelines, the throughput per pipeline is reduced. The used logging backend or performance characteristics of the output plugin might limit the throughput earlier.
```

Comment:
```suggestion
Each Fluent Bit Pod can process up to 10 MB/s of logs for a single LogPipeline. With multiple pipelines, the throughput per pipeline is reduced. The used logging backend or performance characteristics of the output plugin might limit the throughput earlier.
```
If it's the Kubernetes resource, we use CamelCase. Otherwise, keep natural language.